### PR TITLE
[CP-2250] Mudita Center application Start screen update

### PR DESCRIPTION
**Pull request by dkarski**
JIRA Reference: [CP-2250]

### :memo: Description ️

-

### :muscle: Checklist before requesting a review - nice to have

- getState function in async thunk actions is correctly typed
- redux selectors are used in components / prop drilling is reduce

### :exclamation: Checklist before merging a pull request

- change went through the QA process
- translations are updated in dedicated application
- [CHANGELOG.md](./CHANGELOG.md) is updated

[CP-2250]: https://appnroll.atlassian.net/browse/CP-2250?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/core/onboarding/components/onboarding-welcome.styled.tsx [hunk 1]
```diff
@@ -3,49 +3,87 @@
...
+  grid-area: Content;
+  justify-content: space-between;
+  background-color: ${backgroundColor("main")};
+`
 
-    p {
-      font-weight: ${fontWeight("light")};
-    }
-  }
+export const ContentTop = styled.div`
```
Comment (lkowalczyk87) [inline libs/core/onboarding/components/onboarding-welcome.styled.tsx:38]: `header` looks more correct for this case

File: libs/core/onboarding/components/onboarding-welcome.styled.tsx [hunk 2]
```diff
@@ -3,49 +3,87 @@
...
+  line-height: 4.8rem;
+  margin-bottom: 0.4rem;
+`
+export const SubheaderTitle = styled(Text)`
+  font-size: 2.2rem;
+  font-weight: ${fontWeight("light")};
+  color: ${textColor("info")};
+`
+
+export const ContentBottom = styled.div``
```
Comment (lkowalczyk87) [inline libs/core/onboarding/components/onboarding-welcome.styled.tsx:55]: this one can be removed and replaced by `main` tag in`OnboardingWelcome` component

File: libs/core/onboarding/components/onboarding-welcome.styled.tsx [hunk 3]
```diff
@@ -3,49 +3,87 @@
...
-export const WelcomeButton = styled(ButtonComponent)`
-  margin-top: 1.6rem;
+export const ImageWrapper = styled.div`
+  ${baseContainerStyles};
+  max-width: 99rem;
+  justify-content: center;
+  margin-bottom: 2rem;
+`
+
+export const ButtonsContainer = styled.div`
```
Comment (lkowalczyk87) [inline libs/core/onboarding/components/onboarding-welcome.styled.tsx:77]: `footer` looks more correct for this case

Comment (dkarski) [reply libs/core/onboarding/components/onboarding-welcome.styled.tsx:75]: https://github.com/mudita/mudita-center/pull/1730/commits/231f291e962dfb0ea24f10b6921ad72a72728d91

File: libs/core/onboarding/components/onboarding-welcome.styled.tsx (hunk 1 above)
Comment (dkarski) [reply libs/core/onboarding/components/onboarding-welcome.styled.tsx:38]: https://github.com/mudita/mudita-center/pull/1730/commits/231f291e962dfb0ea24f10b6921ad72a72728d91

**Pull request merged**